longest_common_subsequence/Longest_common_subsequence.java

```diff
@@ -0,0 +1,27 @@
+import java.util.*;
```

Comment:
Try to use only those imports which are relevant to the code. Don't unnecessary add the compile time.